components/Features/constants.ts

```diff
@@ -0,0 +1,73 @@
+import { Icon, Linkedin, Slack } from 'react-feather';
+
+interface IIcon {
+  id: number;
+  component: Icon;
+}
+
+export const Icons: Array<IIcon> = [
+  {
+    id: 1,
+    component: Slack
```

Comment:
When you replace icons with svg. You can return these brand logos like `Icon: () => <JavaScriptLogo/>`